OpenRA.Mods.Common/UtilityCommands/LegacyMapImporter.cs

```diff
@@ -393,7 +393,7 @@ void LoadActors(IniFile file, string section)
 						players.Add(parts[0]);
 
 					var loc = Exts.ParseIntegerInvariant(parts[3]);
-					var health = float.Parse(parts[2], NumberFormatInfo.InvariantInfo) / 256;
+					var health = int.Parse(parts[2], NumberFormatInfo.InvariantInfo) * 100 / 256;
```

Comment:
I don't remember how the legacy maps define health, but if it is as a float then you will need to keep this parsing as a float, and then cast to an int when you're done.